Overhaul of Adapter Development Contributing Docs (#1727)
fishtown-analytics/docs.getdbt.com: website/docs/docs/contributing/adapter-development/5-promoting-a-new-adapter.md

- Particular or unique use cases or functionality unlocked by this adapter
- Plans for future / ongoing support / development
- The link to the documentation for using the adapter on the dbt Labs docs site
- An announcement blog

## Is there an example of how to announce new releases of an existing adapter?

This can vary substantially depending on the nature of the release but a good baseline is the types of release messages that [we put out in the #dbt-releases](https://getdbt.slack.com/archives/C37J8BQEL/p1651242161526509) channel.

![Full Release Post](/img/adapter-guide/0-full-release-notes.png)

Breaking this down:

- Visually distinctive announcement - make it clear this is a release
    ![title](/img/adapter-guide/1-announcement.png)
- Short written description of what is in the release
    ![description](/img/adapter-guide/2-short-description.png)
- Links to additional resources
    ![more resources](/img/adapter-guide/3-additional-resources.png)
- Implementation instructions:
    ![installation](/img/adapter-guide/4-installation.png)
- Future plans
    ![coming soon](/img/adapter-guide/5-coming-up.png)
- Contributor recognition (if applicable)

>     ![thank yous](/img/adapter-guide/6-thank-contribs.png)

Author: ```suggestion
- Visually distinctive announcement - make it clear this is a release
    <Lightbox src="/img/adapter-guide/1-announcement.png" title="title"/>
- Short written description of what is in the release
    <Lightbox src="/img/adapter-guide/2-short-description.png" title="description"/>
- Links to additional resources
   <Lightbox src="/img/adapter-guide/3-additional-resources.png" title="more resources"/>
- Implementation instructions:
    <Lightbox src="/img/adapter-guide/4-installation.png" title="more installation"/>
- Future plans
    <Lightbox src="/img/adapter-guide/5-coming-up.png" title="coming soon"/>
- Contributor recognition (if applicable)
    <Lightbox src="/img/adapter-guide/6-thank-contribs.png" title="thank yous"/>
```

---
title: "Promoting a new adapter"
id: "5-promoting-a-new-adapter"
---

## Model for engagement in the dbt community

The most important thing here is recognizing that people are successful in the community when they join, first and foremost, to engage authentically. 

What does authentic engagement look like? It’s challenging to define explicit rules. One good rule of thumb is to treat people with dignity and respect. 

> Contributors to the community should think of contribution *as the end itself,* not a means toward other business KPIs (leads, community members, etc.). Profits are exhausted. Some ways to know if you’re authentically engaging:

Reviewer: ```suggestion
Contributors to the community should think of contribution *as the end itself,* not a means toward other business KPIs (leads, community members, etc.). [We believe that profits are exhaust.](https://www.getdbt.com/dbt-labs/values/#:~:text=Profits%20are%20exhaust.) Some ways to know if you’re authentically engaging:
```

## Who should join the dbt community slack

### People who have insight into what it means to do hands-on [analytics engineering](https://www.getdbt.com/analytics-engineering/) work

The dbt Community Slack workspace is fundamentally a place for analytics practitioners to interact with each other &mdash; the closer the users are in the community to actual data/analytics engineering work, the more natural their engagement will be (leading to better outcomes for partners and the community).

### DevRel practitioners with strong focus

DevRel practitioners often have a strong analytics background and a good understanding of the community. It’s essential to be sure they are focused on *contributing,* not on driving community metrics for partner org (such as signing people up for their slack or events). The metrics will rise naturally through authentic engagement.

### Founder and executives who are interested in directly engaging with the community

This is either incredibly successful or not at all depending on the profile of the founder. Typically

### Software Engineers at partner products that are building and supporting integrations with either dbt Core or dbt Cloud

> This is successful when the engineers are familiar with dbt as a product or at least have taken our training course

Reviewer 2: It seems like these sections were started but never finished.  Is there more here or should we remove them?

Author: @jasnonaz did you or I start these thought? I can't remember...

Author: @matthewshaver better?
```suggestion
### Founder and executives who are interested in directly engaging with the community

This is either incredibly successful or not at all depending on the profile of the founder. Typically, this works best when the founder has a practitioner-level of technical understanding and is interested in joining not to promote, but to learn and hear from users.

### Software Engineers at partner products that are building and supporting integrations with either dbt Core or dbt Cloud

This is successful when the engineers are familiar with dbt as a product or at least have taken our training course. The Slack is often a place where end-user questions and feedback is initially shared, so it is recommended that someone technical from the team be present. There are also a handful of channels aimed at those building integrations, which tend to be a font of knowledge.
```

Reviewer 2: This is great.  Thank you for updating it.
